src/store.js

```diff
@@ -191,6 +191,23 @@ class HybridStore extends MemoryStore {
       STORE.removeItem(REGISTERED_NANO_CONTRACTS_KEY);
     }
   }
+
+  /**
+   * Update nano contract registered address.
+   * @param {string} ncId Nano Contract ID.
+   * @param {string} address Nano Contract registered address.
+   * @async
+   * @returns {Promise<void>}
+   */
+  async updateNanoContractRegisteredAddress(ncId, address) {
+    await super.updateNanoContractRegisteredAddress();
+    const contracts = STORE.getItem(REGISTERED_NANO_CONTRACTS_KEY) || {};
+    const contract = contracts[ncId];
```

Comment:
You can use `this. getNanoContract(ncId)`

Reply:
Ok. I refactored the method to use the store's methods.